Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -359,7 +359,8 @@ void TBWidget::AddChildRelative(TBWidget *child, WIDGET_Z_REL z, TBWidget *refer
 
 void TBWidget::RemoveChild(TBWidget *child, WIDGET_INVOKE_INFO info)
 {
-    assert(child->m_parent);
+    if (!child->m_parent)
+        return;
 
```

Comment:
Ok, I fixed the crash in https://github.com/AtomicGameEngine/AtomicGameEngine/pull/1218

Kind of esoteric, though when a widget is deleted in its own callback, it needs to return true to let the underlying system know the event was handled.  I also changed the remove child guard to ensure ```m_parent == this``` and added // ATOMIC BEGIN // ATOMIC END block with a note on change, whew :)